tools/pot/openvino/tools/pot/algorithms/quantization/utils.py

```diff
@@ -337,6 +337,13 @@ def get_stat_name_by_config(config, stat_type):
     return '_'.join(name_list)
 
 
+def get_input_shape_for_bias(activations_statistics, input_node_name):
+    input_shape = list(activations_statistics[input_node_name]['shape'][0])
```

Comment:
LGTM, but I'm not sure that it's a general case to calculate shape based on the first element. Are there any cases when statistics shapes can differ from sample to sample? I mean such cases, for example, as the `east` model. Has this model been tested, @IndiraSalyahova?
cc @AlexKoff88

Reply:
Valid point. Can we reconsider this and take a mode value (most frequent shape) from the list?
It can be done via scipy as follows:
```
from scipy import stats
stats.mode(activations_statistics[input_node_name]['shape'])[0]
```

Reply:
I am also ok with the changes mostly. Can you try the changes above?
Nighly run is required to merge. Please also test this on buggy models reported in the corresponded tickets.